src/main/groovy/edu/oregonstate/mist/locations/frontend/db/LocationDAO.groovy

```diff
@@ -46,7 +46,7 @@ class LocationDAO {
      */
     String getById(String id) {
         try {
-            return "${ESFullUrl}/${id}/_source".toURL().text
+            return "${ESFullUrl}/${id?.toLowerCase()}/_source".toURL().text
```

Comment:
The Resource class checks that the parameter is not null. This class won't be called with a null id. If for some reason the id is null, the "?." null safe operator takes care of that. 

This method actually makes an http request . This is the short notation. In groovy, you can convert a string to a url object. Calling "text" makes an http request and returns the string response.